src/scripts.js

```diff
@@ -29,8 +29,7 @@ let chosenDate = "2019/07/16" // default date
 const userHydration = new UserHydration(hydrationData);
 const userSleep = new UserSleep(sleepData)
 
-window.addEventListener('load', (event) => {
-  event.preventDefault()
```

Comment:
Turns out we only need `event.preventDefault()` when we're dealing with a form or other semantic tag.